packages/vue/src/components/atoms/SfCircleIcon/SfCircleIcon.html

```diff
@@ -1,6 +1,12 @@
 <SfButton class="sf-circle-icon" v-on="$listeners">
```

Comment:
Can you add the `aria-label` here, and `aria-hidden` to be `true` in SfIcon instead?

Reply:
Sure thing. Regarding your previous comment, should a `role=button` be applied to the component?

Reply:
No need @tohanian . `<button>` by default has a role of button, didn't see we used `SfButton` before  when I made the comment :)

Reply:
@tohanian any update?

Reply:
@aniamusial please finish this issue 👍

Reply:
@tohanian I'll finish this one :) Thank you!

Reply:
https://github.com/DivanteLtd/storefront-ui/pull/492